internal/providers/terraform/azure/testdata/application_insights_test/application_insights_test.golden

```diff
@@ -0,0 +1,16 @@
+
+ Name                                           Monthly Qty  Unit            Monthly Cost 
+                                                                                          
+ azurerm_application_insights.non_usage                                                   
+ └─ Data ingested                             Monthly cost depends on usage: $2.30 per GB 
+                                                                                          
+ azurerm_application_insights.usage                                                       
+ └─ Data ingested                                     1,000  GB                 $2,300.00 
+                                                                                          
+ azurerm_application_insights.with_retention                                              
+ ├─ Data ingested                                     1,000  GB                 $2,300.00 
+ └─ Data retention (30 days)                         30,000  GB                 $3,000.00 
```

Comment:
1. I think this should show the total data retention days, not just the charged part:  `Data retention (120 days)`
2. The quantity seems too high since retention is build in GB-months.  Since it's one month of retention it should be `quantity == 1000GB, cost == $100`.  I clarified my comment on the issue since I'm the one who misunderstood it in the first place :)